index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>My First In Class Assignment</title>
+  <link rel="stylesheet" href="styles/style.css">
+
+  <header class="main-header">
+    <div class="logo">
+    </div>
+
+    <nav class="main-nav">
+    </nav>
+  </header>
```

Comment:
As you might have already realized, markup content, ie. `div`, `section`, `header`, etc. essentially anything that contains content you want people to see, won't be displayed if you put it in the `<head>` element. So this `<header>` element should be removed.